Makefile

```diff
@@ -764,3 +764,32 @@ include tests/integration/tests.mk
 .PHONY: benchcheck
 benchcheck:
 	bin/perfcheck.sh
+
+#-----------------------------------------------------------------------------
+# Target: istio-ecosystem installer
+#-----------------------------------------------------------------------------
+
+export ISTIO_INSTALLER?=${GOPATH}/src/github.com/istio/installer
+INSTALLER_OPTS="-f install/kubernetes/helm/istio/test-values/values-installer-e2e.yaml --set global.istioNamespace=istio-system --set global.hub=${HUB} --set global.tag=${TAG}"
+INSTALLER_REPOSRC=git://github.com/istio/installer.git
+
+force: ;
+
+${GOPATH}/src/github.com/istio/installer: force
+	mkdir -p ${GOPATH}/src/istio-ecosystem
```

Comment:
This is now outdated, we got it moved into istio repo

Reply:
fixed.